packages/replay-next/components/errors/SupportForm.tsx

```diff
@@ -1,40 +1,30 @@
-import { ReactNode, useState } from "react";
+import { ReactNode, useContext, useState } from "react";
 
 import { ModalFrame } from "replay-next/components/errors/ModalFrame";
 import { submitSupportForm } from "replay-next/components/errors/submitSupportForm";
-import { useSessionId } from "replay-next/components/errors/useSessionId";
-import { ReplayClientInterface } from "shared/client/types";
+import { SessionContext } from "replay-next/src/contexts/SessionContext";
 
+import { State } from "./SupportContext";
 import styles from "./SupportForm.module.css";
 
-export function SupportForm({
-  currentUserEmail,
-  currentUserId,
-  currentUserName,
-  onDismiss,
-  placeholder = "Bugs, suggestions, questions are all welcome!",
-  replayClient,
-  title = "Support",
-}: {
-  currentUserEmail: string | null;
-  currentUserId: string | null;
-  currentUserName: string | null;
-  onDismiss: () => void;
-  placeholder?: string;
-  replayClient: ReplayClientInterface;
-  title?: ReactNode;
-}) {
```

Comment:
There was no reason for us to pass these in as props. They can be retrieved from the session context directly.